src/api/messenger.js

```diff
@@ -26,6 +28,7 @@ export async function apiCall (destinationUrl, dataToSend = {}, relocationPage =
 
   const formData = new FormData();
   Object.keys(data).map((k) => formData.append(k, data[k]));
+  formData.append("language", lang); 
```

Comment:
/
```suggestion
  formData.append("__user_language", lang);
```
The api expects `__user_language` as key